### PR TITLE
install.sh complaining about missing directory

### PR DESCRIPTION
**Pull request by probabble**

### CODE REVIEW

**Comment by hburgund** [comment]
Ha! We discovered this issue a few days ago, but you beat us to the punch with committing the fix. Thanks for the contribution.

I believe there is one additional line of code needed to fully remedy the issue, resulting in this block:
```
mkdir $WWW_PATH/settings
cp $SOURCE_PATH/files/var-www-roundware-settings.py $WWW_PATH/settings/roundware_production.py
chown $USERNAME:$USERNAME -R $WWW_PATH/settings
```
If you could give this a corroborative test and if all works on a fresh vagrant build, then just update your pull request and I'm merge it.
Oh, and welcome to the project(!)

**Comment by eosrei** [comment]
Please update(rebase) the commit message with why/what the purpose of the change is.

Thanks for your help!

**Comment by probabble** [comment]
Sorry about the initial commit message - will be more precise in the future... Vagrant builds without errors. 

Not sure if you use the Django test suite, but if I login to the VM and do `manage.py test` I get an error:
Creating test database for alias 'default'...
```
/var/www/roundware/source/roundware/manage.py test
Creating test database for alias 'default'...
Got an error creating the test database: (1044, "Access denied for user 'round'@'localhost' to database 'test_roundware'")
Type 'yes' if you would like to try deleting the test database 'test_roundware', or 'no' to cancel: yes
Destroying old test database 'default'...
Got an error recreating the test database: (1044, "Access denied for user 'round'@'localhost' to database 'test_roundware'")
```
Can make a separate issue for that or ignore it.

Do you all have a talk page / roadmap where I can get an idea of your current goals? I'm really interested in this project - want to use it for an upcoming art installation and hopefully contribute :)

**Comment by eosrei** [comment]
@probabble Many of the todos are listed here: https://github.com/roundware/roundware-server/labels/enhancement We don't have a roadmap document yet. There's still a lot to do, as you can see. @hburgund is working to finalize the contributor agreement.

Run tests with:
```
cd scripts
./tests.sh
```

**Comment by eosrei** [comment]
@probabble Please rebase this branch into a single commit:
```
git rebase -i 44c99ad2fab91f38b52f9023e5578eb3dd92b27b
```

File: install.sh [hunk 1]
```diff
@@ -98,7 +98,9 @@ mkdir -p $STATIC_PATH
 cp $SOURCE_PATH/files/rw_test_audio1.wav $MEDIA_PATH
 
 # Copy default production settings file
+mkdir $WWW_PATH/settings
```
Comment (eosrei) [inline install.sh]: We need to the `-p` in the mkdir command so that running install.sh a second time will not error.

File: install.sh [hunk 2]
```diff
@@ -98,7 +98,9 @@ mkdir -p $STATIC_PATH
...
 cp $SOURCE_PATH/files/var-www-roundware-settings.py $WWW_PATH/settings/roundware_production.py
+chown $USERNAME:$USERNAME -R $WWW_PATH/settings
```
Comment (eosrei) [inline install.sh]: Hmm... this line is redundant due to: https://github.com/roundware/roundware-server/blob/c0d5cf6f7aef56172b3abb0837997a4a1b89fd79/deploy.sh#L57

**Comment by eosrei** [comment]
@probabble Do you mind if I just make a separate commit for the one line change required to fix this issue? I'll set you as commit author.

**Comment by probabble** [comment]
Yes, @eosrei, please do. I'm apparently not skilled at rewriting history :palm_tree:

**Comment by eosrei** [comment]
This issue will be corrected when #217 is merged.

**Comment by probabble** [comment]
:+1: 
super.

**Pull request closed without merging**